.lighthouse/jenkins-x/pullrequest.yaml

```diff
@@ -26,11 +26,13 @@ spec:
           resources: {}
         - name: build-make-linux
           resources: {}
+        - name: make-lint
```

Comment:
add a separate file `lint.yaml`, see: https://github.com/jenkins-x-plugins/jx-secret/blob/main/.lighthouse/jenkins-x/lint.yaml, also check the triggers file.

Reply:
Done, pushing them now

Reply:
remove the step from this file, as we already have a separate lint.yaml file.